Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[workspace]
+members = [
+  "curves/bn254",
+  "models",
+  "test-utils",
+]
+resolver = "2"
+
+[workspace.package]
+version = "0.4.2"
+authors = ["Horizen Labs <admin@horizenlabs.io>"]
+repository = "https://github.com/DimitrisPapac/accelerated-bn-cryptography.git"   # "https://github.com/zkVerify/accelerated-bn-cryptography"
+homepage = "https://horizenlabs.io"
+keywords = ["cryptography", "elliptic-curves", "pairing"]
+categories = ["cryptography"]
+include = ["Cargo.toml", "src", "README.md", "LICENSE-APACHE", "LICENSE-MIT"]
+license = "MIT/Apache-2.0"
+edition = "2021"
+
+[workspace.dependencies]
+ark-ec = { version = "0.4.0", default-features = false }
+ark-ff = { version = "0.4.0", default-features = false }
+ark-std = { version = "0.4.0", default-features = false }
+ark-serialize = { version = "0.4.0", default-features = false }
+ark-bn254 = { version = "0.4.0", default-features = false, features = ["curve"] }
+ark-scale = { version = "0.0.12", default-features = false, features = ["hazmat"] }
```

Comment:
Is the feature `hazmat` required?

Reply:
Not in this repository. However, the `accelerated_bn` module within zkVerify does make use of it. Specifically:
`type ArkScaleProjective<T> = ark_scale::hazmat::ArkScaleProjective<T>;`